OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -313,7 +313,7 @@ public HackyAI(HackyAIInfo info, ActorInitializer init)
 			claimLayer = World.WorldActor.TraitOrDefault<ResourceClaimLayer>();
 			pathfinder = World.WorldActor.Trait<IPathFinder>();
 
-			isEnemyUnit = unit =>
+			IsEnemyUnit = unit =>
 				Player.Stances[unit.Owner] == Stance.Enemy
 					&& !unit.Info.HasTraitInfo<HuskInfo>()
```

Comment:
I only found !Disposed && !IsDead in GroundStates and IsDead || Disposed in DevCommands, but that's still worth fixing (at least in GroundStates).